tests/pubsub/utils.py

```diff
@@ -1,12 +1,26 @@
 import uuid
+import struct
 from libp2p.pubsub.pb import rpc_pb2
 
-def generate_message_id():
+
+def message_id_generator(start_val):
     """
     Generate a unique message id
-    :return: messgae id
+    :param start_val: value to start generating messages at
+    :return: message id
     """
-    return str(uuid.uuid1())
+    val = start_val
+    def generator():
```

Comment:
I followed this [here](https://stackoverflow.com/questions/846038/convert-a-python-int-into-a-big-endian-string-of-bytes)

Reply:
@robzajac is that right?

Reply:
Actually if you want 64-bit I think the formatting character should be `q` or `Q` instead of `I` based on the table here (documentation for `struct`): https://docs.python.org/3/library/struct.html#format-characters

I suppose unsigned makes sense since message IDs are probably nonnegative.

Reply:
`Q` LGTM, 64-bit big-endian uint

Reply:
Fixed.